packages/dao/contracts/Api3BaseTemplate.sol

```diff
@@ -19,6 +19,8 @@ import "@aragon/os/contracts/factory/DAOFactory.sol";
 import "@aragon/os/contracts/common/IsContract.sol";
 import "@aragon/os/contracts/common/Uint256Helpers.sol";
 import "@aragon/id/contracts/IFIFSResolvingRegistrar.sol";
+//TODO: This is added to use this contract in tests, I think this is stupid, need to think ho to change it
```

Comment:
we can add `test/Imports.sol` and import it there

Reply:
To be honest, that's kind of crutch as well, but it is definitely a better soution...